packages/components/src/panel/style.scss

```diff
@@ -77,6 +77,7 @@
 .components-panel__body-toggle.components-button {
 	position: relative;
 	padding: $grid-unit-20;
+	padding-right: 48px;
```

Comment:
Thank you for your contribution @MatzeKitt !

I think you should make your change in the above `padding` rule. Also it's better to use [GB declared variables](https://github.com/WordPress/gutenberg/blob/4e9aa5dc71d8bdef4ee7b7165d8a8704e28ef657/packages/base-styles/_variables.scss#L39).
`$grid-unit-60` equals `48px`.

Reply:
I changed my pull request accordingly. 🙂